Components/Image/image.css

```diff
@@ -1,10 +1,13 @@
 
 .responsive-image {
-    min-width: 100%;
```

Comment:
For responsive image you have to add max-inline size, block size , object-fit properties. For more reference refer to Vikram CSS Basic class 3.

Reply:
okay